core/nuxeo-connection.js

```diff
@@ -133,96 +296,42 @@ import './nuxeo-element.js';
      * Connects nuxeo client.
      */
     connect() {
-      // Create the client if needed
-      // If we already have a client with the same connection info just keep it
-      const id = this.connectionId ? this.connectionId : Object.keys(nxClients)[0];
-      this.client = nxClients[id];
-      if (this.client) {
-        // if this instance does not have any properties set properties from the client
-        if (!this.url && !this.username && !this.password && !this.repositoryName) {
-          this.set('url', this.client._baseURL);
-          this.username = this.client._username;
-          this.password = this.client._password;
-          this.repositoryName = this.client._baseOptions.repositoryName;
-        }
-        // if properties match the existing client use it
-        if (
-          this.client._baseURL === this.url &&
-          this.client._username === this.username &&
-          this.client._password === this.password &&
-          this.client._baseOptions.repositoryName === this.repositoryName
-        ) {
-          // return the stored connection request promise and chain _handleConnected to update instance properties
-          return this.client._promise.then(this._handleConnected.bind(this));
-        }
-        // otherwise override the client with the new properties
-        this.client = null;
-      }
-      const options = {
-        baseURL: this.url,
-        schemas: ['*'],
-      };
+      this.connection.connectionId = this.connectionId;
```

Comment:
We inherit the `this.connection` from the `ConnectionMixin`, right?
But we don't need to declare `this.connectionId` and `this.url`, `this.repositoryName`, etc.?